src/mono/wasm/build/ILLink.Substitutions.NoWasmIntrinsics.xml

```diff
@@ -9,5 +9,14 @@
     <type fullname="System.Runtime.Intrinsics.Wasm.PackedSimd">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Runtime.Intrinsics.X86.X86Base">
```

Comment:
These are already in ILLink.Substitutions.NoX86Intrinsics.xml, why is this needed ?

Reply:
It is not needed, I forgot to remove it from my commit :-)